moduleroot/.github/PULL_REQUEST_TEMPLATE.md.erb

```diff
@@ -0,0 +1,12 @@
+Please ensure that you have completed all of the following steps:
+
+- [ ] Provided your PR with a meaningful title as this will become an entry in the `CHANGELOG.md`
+- [ ] Documented your change in the `README.md` if necessary
+- [ ] Included tests for your functionality
+- [ ] Added a label to categorize your change
+- [ ] Confirmed that all tests are passing
+
+This is an example of a great [PR](https://git.io/fpRgw). For additional information please see our [contributing guidelines](https://puppet.com/docs/puppet/latest/contributing.html).
+Someone from Puppet will review your PR as soon as possible.
+
+**Thank you for taking the time to contribute to our supported modules.**
```

Comment:
This is the default template for ALL PDK installations. Specifically mentioning `our supported modules` does not make sense for non Puppetlabs or unsupported modules